spk/sabnzbd/Makefile

```diff
@@ -14,7 +14,34 @@ DESCRIPTION_SPN = SABnzbd hace que Usenet sea lo más simple posible, automatiza
 ADMIN_PORT = 8080
 RELOAD_UI = yes
 DISPLAY_NAME = SABnzbd
-CHANGELOG = "1. Update to 1.0.0<br>2. Warning: When you upgrade an existing 0.7.x installation, release 1.0.0 will NOT be able to finish the queue. It's best to empty your queue first using 0.7.x. Alternatively, you can install 1.0.0 and then use queue repair on the Status page. Some information (like the order of the queue) will be lost."
+CHANGELOG = "1. Update to 1.0.0<br>2. Warning: When you upgrade an existing 0.7.x installation, release 1.0.0 will NOT be able to finish the queue. It's best to empty your queue first using 0.7.x. Alternatively, you can install 1.0.0 and then use queue repair on the Status page. Some information (like the order of the queue) will be lost.
```

Comment:
Please leave the changelog as-is (although you should update the version to what it should be, 1.0.3, not 1.0.0 at this point)

Package Center can't handle the formatting, you'd need to convert it to html for it to work properly. That's why we generally put just the minimum amount of info in there: it's a package changelog, not a software changelog. If people want to see the changes in Sabnzbd itself, they can look it up on the sabnzbd website.